.github/workflows/github-ci.yml

```diff
@@ -107,24 +107,25 @@ jobs:
 
   build_production:
     # Build Vlasiator witth production flags
-    runs-on: ubuntu-latest
-    container: ursg/vlasiator_ci:20230220_1
+    runs-on: carrington
+      #container: ursg/vlasiator_ci:20230220_1
     needs: build_libraries
 
     steps:
     - name: Checkout source
       uses: actions/checkout@v3 
-    - name: Download libraries
-      uses: actions/download-artifact@v3
-      with:
-        name: libraries
-    - name: Unpack libraries
-      run: tar --zstd -xvf libraries.tar.zstd
-    - uses: ursg/gcc-problem-matcher@master
+      with: 
+        submodules: true
```

Comment:
So is this now a temporary set-up, as you've only declared the fsgrid submodule, and all the other libraries come from the standard carrington_gcc_openmpi makefile? I guess the way ahead would be to set up the libraries tar file for those libraries which we don't control ourselves (e.g. vectorclass, zoltan, whatever) and set up the fmihpc libraries as submodules?

Reply:
Yup. We could also point submodules to commits on their respective repos (looks like everything is available via git*), or fork those under fmihpc. Besides, forking vectorclass would be handy to 'pre-install' the plugin header in a commit of our own.

Reply:
Hmm, that is indeed an interesting idea. It'll also be less of a moving target for us to track then.
(On the other hand, if there are ever any important updates for some new architecture, and we only update *then*, we might have a whole bunch of things to fix on our hands)